edb.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=edb
-GenericName=edb debugger
-Comment=edb debugger 
+GenericName="Evan's Debugger"
```

Comment:
I'm sorry for not looking at this closer before telling you to submit your fixes. According to XDG spec, 'GenericName' is a generic description of applications class, so I think a plain 'Debugger' would be more correct. `Evan's Debugger` would be more correct for 'Name' key.

https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys

Reply:
No worries it's an easy fix